org.eclipse.winery.topologymodeler.ui/src/app/winery.component.ts

```diff
@@ -72,6 +72,8 @@ export class WineryComponent implements OnInit {
         // Grouped NodeTypes
         this.backendService.groupedNodeTypes$.subscribe(JSON => {
             this.initEntityType(JSON, 'groupedNodeTypes');
+            this.loaded = {loadedData: true, generatedReduxState: false};
```

Comment:
Could you implement a method which waits for **every** obersvable to return?

Reply:
done